libraries/profile.rb

```diff
@@ -76,7 +76,7 @@ class ComplianceProfile < Chef::Resource # rubocop:disable Metrics/ClassLength
         end
       end
 
-      FileUtils.move(tf.path, path) unless tf.nil?
+      FileUtils.cp(tf.path, path) unless tf.nil?
```

Comment:
Should we delete content from `tf.path`? Please add a comment, why we use cp instead of mv.